scr/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="../style/style.css">
+    <title>Hello MicroVerse</title>
+</head>
+<body>
+    <h1>Hello MicroVerse..!</h1>

```

Comment:
- [ ] It is required to give the `h1` tag a class name and use that class name to style the content of the `h1` tag.

### Project Requirement.
![project-rqtm](https://user-images.githubusercontent.com/26597432/235604550-d77a246f-da4b-412c-a19e-0ae59ca2d152.png)